### ISSUE TITLE
CustomParameters are sorted and do not support UTF-8 encoding

### ISSUE THREAD

**Issue by rancsik** [trivago/cluecumber-report-plugin#157]
**Describe the bug**
CustomParameters (tags) are displayed in natural order (in Alphabetical order) and do not support UTF-8 encoding. Not in Tag names and nor in value.

**To Reproduce**
Steps to reproduce the behaviour:
1. Apply CustomParameters in pom.xml
e.g.:

```
 <customParameters>
                        <Zweck_des_Dokuments>Testprotokoll zur Qualitätssicherung</Zweck_des_Dokuments>
                        <Gültig_fuer>Software QS</Gültig_fuer>
                        <Verantwortlich>Somebody</Verantwortlich>
                        <Klassifikation>Intern</Klassifikation>
</customParameters>

```
2. Genarete report with maven
3. Open genarated index.html file
4. Check the CustomParameters  part in top of the document
5. Check the Tag names are sorted and do not support UTF-8 encoding.
Displayed:

```
G�ltig fuer: | Software QS
Klassifikation: | Intern
Verantwortlich: | Somebody
Zweck des Dokuments: | Testprotokoll zur Qualit�tssicherung

```
**Expected behavior**
customParameters are displayed in defined order and supports UTF-8 encoding . 

**Attachments**
![report](https://user-images.githubusercontent.com/4921990/56145116-f10d4f00-5fa3-11e9-8666-72dfa43a6ce8.JPG)

**Comment by laxersaz** [comment]
I just verified parts of your bug.
The order is indeed wrong. However, the encoding is correct in my case:
![image](https://user-images.githubusercontent.com/5775857/56200339-e0f67d80-603e-11e9-9c1b-a23b1dec9c6a.png)

Do you have this set in your POM?
```
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

**Comment by laxersaz** [comment]
Apparently, Maven itself orders those parameters before passing them to the plugin. So unfortunately there is no easy solution at this point.

**Comment by rancsik** [comment]
Hi, 
thank you for quick answer. Both parameters are set in my pom xml under properties tag, but anyway the tag names are displayed incorrectly. But in other part of my report, there are also some extra characters like öäüß and they are displayed correctly…
I am using apache-maven-3.6.1 under windows environment. I also tried to set -Dfile.encoding=UTF-8 but without any success. 
In STD OUT logs I can see, that during the generation the property is displayd correctly, but at the end it is printed in html like: <strong>G��g f��/strong>
STD Output:
- custom parameter                           : Gültig_für -> Software QS
Best regards,
Istvan

**Comment by laxersaz** [comment]
Thanks for the additional information. Will look into it.

**Comment by laxersaz** [comment]
I found a way to keep the passed parameters sorted. This will be available in Cluecumber 2.0.0 which is in development. However, the encoding issue is still under investigation

![image](https://user-images.githubusercontent.com/5775857/59288482-2c5e8e00-8c74-11e9-87de-d47655ac6a58.png)

**Comment by laxersaz** [comment]
Added more encoding options so hopefully this should be fixed in 2.0.0
